src/components/Tooltip/Tooltip.tsx

```diff
@@ -44,19 +44,34 @@ const Tooltip = ({
   const [inlineStyles, setInlineStyles] = useState({})
   const [inlineArrowStyles, setInlineArrowStyles] = useState({})
   const [show, setShow] = useState(false)
+  const [rendered, setRendered] = useState(false)
   const wasShowing = useRef(false)
   const [calculatingPosition, setCalculatingPosition] = useState(false)
   const lastFloatPosition = useRef<IPosition | null>(null)
   const { anchorRefs, setActiveAnchor: setProviderActiveAnchor } = useTooltip(id)
   const [activeAnchor, setActiveAnchor] = useState<React.RefObject<HTMLElement>>({ current: null })
   const hoveringTooltip = useRef(false)
 
-  const handleShow = (value: boolean) => {
-    if (setIsOpen) {
-      setIsOpen(value)
-    } else if (isOpen === undefined) {
-      setShow(value)
+  useEffect(() => {
+    if (!show) {
+      setRendered(false)
     }
+  }, [show])
+
+  const handleShow = (value: boolean) => {
+    setRendered(true)
+
+    /**
+     * this `timeout` is necessary because the component
+     * needs to be rendered to calculate the position correctly
+     */
+    setTimeout(() => {
+      if (setIsOpen) {
+        setIsOpen(value)
+      } else if (isOpen === undefined) {
+        setShow(value)
+      }
+    }, 10)
```

Comment:
needs to double-check the delay tooltip

Reply:
Ideally we shouldn't use a timeout, and actually wait for the styles to be updated. 

I couldn't see weird behaviors, so it should be fine for now, but it's something to think about.